lib/pure/memfiles.nim

```diff
@@ -15,6 +15,27 @@
 ## It also provides some fast iterators over lines in text files (or
 ## other "line-like", variable length, delimited records).
 
+runnableExamples("-r:off"):
+  # Example of how to create a memfile and
+  # write data to it and read it again. 
+  import memfiles, os
```

Comment:
```suggestion
  import std/[memfiles, os]
```

but actually see 
https://github.com/nim-lang/Nim/pull/17203
https://github.com/nim-lang/RFCs/issues/352

about including the module's own import in the top-level example